src/css/color.zig

```diff
@@ -10,11 +12,45 @@ pub const Color = struct {
     a: u8,
 
     pub fn rgb(r: u8, g: u8, b: u8) Color {
-        return rgba(r, g, b, 0xFF);
+        return rgba(r, g, b, .{ .value = 1.0 });
     }
 
-    pub fn rgba(r: u8, g: u8, b: u8, a: u8) Color {
-        return .{ .r = r, .g = g, .b = b, .a = a };
+    pub fn rgba(r: u8, g: u8, b: u8, a: NumberOrPercentage) Color {
```

Comment:
I think in this case, the last parameter should be `AlphaValue` (different from `NumberPercentage`) but keep it this way, I can fix it later.